care-o-bot/105dc16/105dc16.bug

```diff
@@ -43,7 +44,10 @@ fix:
     - cob_control/cob_twist_controller/CMakeLists.txt
   languages:
     - CMake
-  time: 2014-11-21 (12:57)
+  time: 2014-11-14T08:16:00Z
+        # fix was released in 0.6.2
+        # git log --date=iso --pretty="%h %ad %cd %s" -- indigo/distribution.yaml | grep cob_control | grep 0.6.2
+        # 2014-12-15T17:49:37+0100
```

Comment:
@gavanderhoorn

Reply:
?

Reply:
Revisited `rosinstall_generator_tm.sh` and it already does this :D. Please ignore.